saleor/checkout/interface.py

```diff
@@ -0,0 +1,33 @@
+from dataclasses import dataclass
+
+from prices import Money, TaxedMoney
+
+
+@dataclass
+class TaxedPricesData:
+    """Store a prices data with applied taxes.
+
+    'price' includes discount from sale if any valid exists.
+    'price_with_voucher' includes voucher discount and sale discount if any valid
+    exists.
+    'undiscounted_price' is a price without any sale and voucher.
+    """
+
+    undiscounted_price: TaxedMoney
+    price_with_voucher: TaxedMoney
+    price: TaxedMoney
+
+
+@dataclass
+class PricesData:
```

Comment:
I was thinking that generic name would be good in case if we will need to use them in the calculation of total or subtotal in the future. Renaming it to `TaxedLinePrice` will force us to create or rename it again for total/subtotal